src/panoramic/cli/command.py

```diff
@@ -46,13 +46,26 @@
 logger = logging.getLogger(__name__)
 
 
+def configure_anonymous_analytics():
+    analytics_opt_out = click.confirm(
+        "\nWith your permission, panoramic-cli can automatically collect \n"
+        "analytical information about usage of the tool\n"
+        "and send it to Panoramic to help improve quality of its products.\n"
+        "No passwords, credentials or personal information will be shared.\n"
+        "\nAllow anonymous usage analytics?",
```

Comment:
Can we replace this with something more simple?

```
Do you want to provide usage metrics to Panoramic? No passwords,
credentials, or personal information will be shared.
```

Reply:
Updated.